data/mods/Xedra_Evolved/items/armor/integrated.json

```
@@ -542,5 +542,33 @@
         "breathability": "SECOND_SKIN"
       }
     ]
+  },
+  {
+    "id": "integrated_bloodbank",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "organic bloodbank" },
+    "description": "You can vaguely sense this organ… somewhere in your body.",
+    "volume": "1 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "flesh" ],
+    "symbol": "x",
+    "color": "magenta",
+    "flags": [ "INTEGRATED", "UNBREAKABLE", "PERSONAL", "NO_SALVAGE", "TRAIDER_AVOID" ],
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "watertight": true,
+        "max_contains_volume": "5 L",
```

```suggestion
        "max_contains_volume": "1 L",
```
1 liter is 20% of the total volume of blood in a human body and by moving it to tier three it also ties in with some organs like your stomach and intestines shrinking as you begin to use blood instead of calories.  Plus 5 liters absolutely negates any reason to ever over stuff yourself with blood.  1 liter feels like a nice top off for emergencies.

---

from what i know people already don't overstuff themselves simply because it's not worth it
orininally it should be just a mutation that allow you to drink blood if you have no container with yourself
i'll change it